sounds/migrations/0008_download_license.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-07-12 12:53
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sounds', '0007_auto_20161018_1809'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='download',
+            name='license',
+            field=models.ForeignKey(blank=True, default=None, null=True, on_delete=django.db.models.deletion.CASCADE, to='sounds.License'),
```

Comment:
after **live** conversation we agreed that we should populate the license field for all downloads. However, because such population could take a lot of time, we propose to proceed as follows:

1. Keep the migration as it is now (creating the column but not populating it)
2. Modify the code in the attribution page to avoid breaking if license info is null (just show the license of the sound)
3. Deploy these changes and then make a script which populates the downloads table. This can do it bit by bit, and it's not a problem if it takes even days to complete.
4. (when the script has finished). Update the code in the attribution page to show the license of the download object.

We should add a note in the migration which explains that the script must be run.